server/common/api.yml

```diff
@@ -1005,6 +1040,36 @@ components:
               example: 2020IPG-000
           example: [[2018IPG-000, 2018IPG-001, 2018IPG-002], [2018IPG-003]]
 
+    SendEventReminderBody:
+      type: object
+      required:
+        - eventRecipients
+        - text
+        - eventName
+        - link
+        - winners
+      properties:
+        eventRecipients:
+          type: array
+          items:
+            description: 2D Array of recipients
+            type: string
+            items:
```

Comment:
The inner items is used when it's a 2D array, we should rather have something like,
```
eventRecipients:
  type: array
  items:
    type: string
    description: Email Recipients
    example: bcs_201831@iiitm.ac.in
  example: ['email1', 'email2', 'email3]
```

Reply:
I think this hasn't been fixed yet, could you please look into it?

Reply:
Could you please re-check because I've made changes in previous commit

Reply:
I did, currently it looks like this:-
```
        eventRecipients:
          type: array
          items:
            description: Array of recipients
            type: string
            items:
              type: string
              description: Email Recipients
              example: bcs_201831@iiitm.ac.in
          example:
            [' bcs_201831@iiitm.ac.in', 'gunashekherproddatoori@gmail.com', 'aasf.iiitmg@gmail.com']
```
whereas it should look like this:-
```
        eventRecipients:
          type: array
          description: Array of recipients
          items:
            type: string
            description: Email Recipients
            example: bcs_201831@iiitm.ac.in
```